app/views/ckey2bibframes/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<h1>Ckey to Bibframe2 Conversion</h1>
```

Comment:
We discussed creating an issue to explore additional validation of the ckey - but did we create that issue already?